arkouda/pdarrayclass.py

```diff
@@ -15,7 +15,7 @@
 
 __all__ = ["pdarray", "info", "clear", "any", "all", "is_sorted", "sum", "prod", 
            "min", "max", "argmin", "argmax", "mean", "var", "std", "mink", 
-           "maxk", "argmink", "argmaxk", "register_pdarray", "attach_pdarray", 
+           "maxk", "argmink", "argmaxk", "attach_pdarray",
            "unregister_pdarray"]
```

Comment:
did you want to remove `unregister_pdarray` also?

Reply:
I thought you mentioned there were times when you needed to manually remove something by name because the python object got mis-placed so I left it in.  I can take this out if you want.

Reply:
you're right I think we will leave the interface i there, for now. Thanks!